cmd/truss/main.go

```diff
@@ -199,20 +198,27 @@ func parseInput() (*truss.Config, error) {
 	}
 
 	// PBGoPackage
-	if *pbPackageFlag == "" {
-		cfg.PBPackage = cfg.ServicePackage
-		cfg.PBPath = cfg.ServicePath
-	} else {
-		p, err := build.Default.Import(*pbPackageFlag, wd, build.FindOnly)
-		if err != nil {
-			return nil, err
-		}
-		if !fileExists(p.Dir) {
-			return nil, errors.Errorf("specified package path for .pb.go output directory does not exist: %q", p.Dir)
-		}
-		cfg.PBPackage = p.ImportPath
-		cfg.PBPath = p.Dir
+	//
+	// It used to be the case that the package where the .pb.go files would be
+	// placed by default was into the newly-generated truss service. However,
+	// we now want to keep our .pb.go files directly next to our .proto files,
+	// so that's where we're going to place them by default. This implies an
+	// assumption that our .proto files exist within our GOPATH.
+	//
+	// Also, if they've passed in multiple protobuf files, we're going to use
```

Comment:
In the future, we want to support passing in several protobuf `packages`. Though only one package will be allowed to have a `Service` definition. 

Is there a way to update this such that `protoDir` is created from the `DefPath` of the `.proto` file that contains the service definition? I know you have that function that tells you what the `ServiceName` is, so I believe that function could be used/modified to figure out what the path to the `.proto` file that contains the service definition.

That make sense? It was a bit wordy.

Reply:
I think this can be added in a future PR.